src/api/Api.js

```diff
@@ -206,20 +206,24 @@ class Api {
      * @param {string} url The url for the request
      * @param {*} data Any data that should be send with the request. This becomes the body of the PUT request.
      * @param {boolean} [useMergeStrategy=false]
-     *
+     * @param {Object.<string, any>} options The request options are passed as options to the fetch request.
+     * 
      * @returns {Promise.<*>} The response body.
      */
     update(url, data, useMergeStrategy = false, options = {}) {
         let payload = data
+
         // Ensure that headers are defined and are treated without case sensitivity
-        options.headers = new Headers(options.headers || {})
+        const requestOptions = {
+            headers: new Headers(options.headers || {})
+        }
```

Comment:
Nice, thanks!  Do we need to pass through `...options` in case other things are defined or is `headers` the only valid option?

Reply:
That was a bit too fast! It's pushed!